bool is3big
bool is4big
bool is5big